src/components/JourneyComparison/JourneyComparison.tsx

```diff
@@ -12,20 +12,26 @@ import FareComparison from "./FareComparison";
 
 import styles from "./JourneyComparison.module.scss";
 
-const getTotalJourneyDuration = (journey: JourneyInfo) => {
+const getTotalJourneyDuration = (journey: JourneyInfo, departAfter: boolean) => {
     if (journey.navigationFailed) {
         return null;
     }
-    const first = journey.segments[0];
+    const first =
```

Comment:
It looks like there are a handful of places where we now do conditional manipulation of the `JourneyInfo` on the client based on the value of `departAfter`. I wonder if it may be cleaner to just remove the first `transfer` segment (the waiting time) before sending the journey back to the client in this function: 

https://github.com/transitmatters/regional-rail-explorer/blob/8cf4da72ab4b20d4179a3b85877e0bc405a746ae/src/server/navigation/index.ts#L8

Then the client wouldn't have to care about the value of `departAfter` here, or in `renderJourneyDuration`, `getStartTime`, etc. This is also where we modify the journey for `reverse = true`, so it feels like it would be well-placed there.